File: FarmCommerce/FarmCommerce.Model/Firma.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FarmCommerce.Model
{
    public class Firma
    {
        public int FirmaId { get; set; }

        public string? Naziv { get; set; }

        public string? Adresa { get; set; }
        public string? Email { get; set; }

        public decimal? Telefon { get; set; }

        public string? SlikaUrl { get; set; }

    }
}

File: FarmCommerce/UI/farmcommerce_admin/lib/models/firma.dart
import 'package:json_annotation/json_annotation.dart';

part 'firma.g.dart';


@JsonSerializable()
class Firma {
  int? firmaId;
  String? naziv;
  String? adresa;
  String? telefon;
  String? email;
  String? slikaUrl;

  Firma(this.firmaId, this.naziv, this.adresa, this.telefon, this.email, this.slikaUrl);

    /// A necessary factory constructor for creating a new User instance
  /// from a map. Pass the map to the generated `_$UserFromJson()` constructor.
  /// The constructor is named after the source class, in this case, User.
  factory Firma.fromJson(Map<String, dynamic> json) => _$FirmaFromJson(json);

  /// `toJson` is the convention for a class to declare support for serialization
  /// to JSON. The implementation simply calls the private, generated
  /// helper method `_$UserToJson`.
  Map<String, dynamic> toJson() => _$FirmaToJson(this);
}